src/tools/concurrency/TraceParser.java

```diff
@@ -75,6 +136,13 @@ private void parseSymbols() {
     }
   }
 
+  private void readSourceSection(final ByteBuffer b) {
```

Comment:
I saw below some comments referring that each getShort call corresponds to the file, start line, start column and char length properties that were recorded, I think is useful to leave these comments